src/SimpleList/index.js

```diff
@@ -45,13 +55,33 @@ export default class SimpleList extends Component {
     )
   }
 
+  debounce = (fn, time) => {
```

Comment:
This implementation is different from the others, but is actually probably better. When you refactor it, use this implementation